bazel/repositories.bzl

```diff
@@ -0,0 +1,21 @@
+"""External dependencies for rosjava_core, excluding Maven dependencies.
+
+Maven dependencies must be added to the workspace with bazel-deps.
+"""
+
+def rosjava_repositories():
+  # DO NOT SUBMIT: update this after the pull request has been merged:
+  # https://github.com/rosjava/rosjava_bootstrap/pull/66
```

Comment:
I've revied the other. Please update this one.

Reply:
Done.